spark/templates/spark-worker-deployment.yaml

```diff
@@ -16,6 +22,10 @@ spec:
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
         component: "{{ .Release.Name }}-{{ .Values.Worker.Component }}"
     spec:
+      volumes:
+        - name: spark-tmp
+          hostPath:
+            path: /data-a/spark-temp-data
```

Comment:
I would place this in a configuration parameter in `values.yaml`

Reply:
Much better, yes. Will do.